### PR TITLE
cs and sh

### PR DESCRIPTION
**Pull request by switch87**
Small changes in some snippets for cs like used in visual studio.

added GPL2 license to sh snippets

### CODE REVIEW

File: snippets/cs.snippets [hunk 1]
```diff
@@ -103,15 +103,15 @@ snippet for
 	}
 # foreach
 snippet fore
-	foreach (var ${1:entry} in ${2}) {
+	foreach (${1:var} ${2:entry} in ${3}) {
```
Comment (lpil) [inline snippets/cs.snippets]: Is it that you want to delete the `var` word sometimes? If so, can you also put the space in the `{}`s please.

**Comment by lpil** [comment]
The sh snippets have nothing to do with sh. Could you remove them please?

File: snippets/cs.snippets (hunk 1 above)
Comment (switch87) [inline snippets/cs.snippets]: no, it is becouse you can use var or specify the variable type, it does not work without it.
like I wrote it it works the same as it does in visual studio.

**Comment by switch87** [comment]
ok, done that.

**Comment by lpil** [comment]
Thanks.

**Pull request merged**